includes/Jobs/GenerateProductFeed.php

```diff
@@ -105,6 +109,10 @@ protected function process_items( array $items, array $args ) {
 	 * @param array $processed_items Array of product fields to write to the feed file.
 	 */
 	protected function write_processed_items_to_feed( $processed_items ) {
+		// Check if we have any items to write.
+		if ( empty( $processed_items ) ) {
```

Comment:
What do you think if we log that `$processed_items` is empty?

Reply:
I don't think we should. This is a normal case in a lot of scenarios.